packages/app/webhook.test.ts

```
@@ -0,0 +1,223 @@
+    })
+      .map(([k, v]) => `${k}=${JSON.stringify(v)}`)
+      .join(" ");
+
+    try {
+      const process = await ezSpawn.async(
+        `pnpm cross-env ${env} pnpm run -w publish:playgrounds`,
+        [],
+        {
+          stdio: "overlapped",
+          shell: true,
+          env: { ...process.env, API_URL: workerUrl },
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)